modules/seqexec/engine/src/main/scala/seqexec/engine/Sequence.scala

```diff
@@ -25,9 +25,7 @@ final case class Sequence(
 
 object Sequence {
 
-  type Id = String
-
-  def empty[A](id: Id, m: SequenceMetadata): Sequence = Sequence(id, m, Nil)
+  def empty[A](id: Observation.Id, m: SequenceMetadata): Sequence = Sequence(id, m, Nil)
```

Comment:
@jluhrs We could make the `Id` a type parameter though we probably would never use anything but `Observation.Id`. WDYT?

Reply:
I cannot think of any case where we would need another type.